webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js

```diff
@@ -20,7 +20,7 @@ const ComponentVersion = ({ componentCV }) => {
   const manualVersionText = (latestVersion === version) ? __('Latest version') : __(`New version is available: Version ${latestVersion}`);
   return (
     <>
-      <Link to={urlBuilder('labs/content_views', '', id)}> Version {version}</Link>
+      <Link to={urlBuilder('labs/content_views', '', id)}> {__('Version')} {version}</Link>
```

Comment:
This will end up as a string like "Version 5.0", so "Version" alone might be hard to translate.  Not sure what to do about it though; should probably mark it for translation anyway.

Reply:
using `sprintf`, although for `Version X` I don't know how many option there are.

Reply:
It's the number of content view versions the user creates; I'd say it's usually between 1 and 20